eth/handler.go

```diff
@@ -97,16 +98,17 @@ type txPool interface {
 // handlerConfig is the collection of initialization parameters to create a full
 // node network handler.
 type handlerConfig struct {
-	NodeID         enode.ID               // P2P node ID used for tx propagation topology
-	Database       ethdb.Database         // Database for direct sync insertions
-	Chain          *core.BlockChain       // Blockchain to serve data from
-	TxPool         txPool                 // Transaction pool to propagate from
-	Network        uint64                 // Network identifier to advertise
-	Sync           ethconfig.SyncMode     // Whether to snap or full sync
-	BloomCache     uint64                 // Megabytes to alloc for snap sync bloom
-	EventMux       *event.TypeMux         // Legacy event mux, deprecate for `feed`
-	RequiredBlocks map[uint64]common.Hash // Hard coded map of required block hashes for sync challenges
-	NoTxGossip     bool                   // Disable P2P transaction gossip
+	NodeID              enode.ID               // P2P node ID used for tx propagation topology
+	Database            ethdb.Database         // Database for direct sync insertions
+	Chain               *core.BlockChain       // Blockchain to serve data from
+	TxPool              txPool                 // Transaction pool to propagate from
+	Network             uint64                 // Network identifier to advertise
+	Sync                ethconfig.SyncMode     // Whether to snap or full sync
+	BloomCache          uint64                 // Megabytes to alloc for snap sync bloom
+	EventMux            *event.TypeMux         // Legacy event mux, deprecate for `feed`
+	RequiredBlocks      map[uint64]common.Hash // Hard coded map of required block hashes for sync challenges
+	NoTxGossip          bool                   // Disable P2P transaction gossip
+	TxGossipNetRestrict *netutil.Netlist       // Restrict tx gossip to specific IP networks
```

Comment:
```suggestion

	// OP Stack additions
	NoTxGossip          bool                   // Disable P2P transaction gossip
	TxGossipNetRestrict *netutil.Netlist       // Restrict tx gossip to specific IP networks
```
This avoids the diff to reformat all other upstream fields.